packages/superset-ui-chart/src/components/SuperChart.tsx

```diff
@@ -54,26 +100,29 @@ export default class SuperChart extends React.PureComponent<Props, {}> {
       disableErrorBoundary,
       FallbackComponent,
       onErrorBoundary,
+      Wrapper = React.Fragment,
       ...rest
     } = this.props as PropsWithDefault;
 
     const chart = (
-      <SuperChartCore
-        ref={this.setRef}
-        id={id}
-        className={className}
-        chartType={chartType}
-        chartProps={this.createChartProps({
-          ...rest,
-          height,
-          width,
-        })}
-        preTransformProps={preTransformProps}
-        overrideTransformProps={overrideTransformProps}
-        postTransformProps={postTransformProps}
-        onRenderSuccess={onRenderSuccess}
-        onRenderFailure={onRenderFailure}
-      />
+      <Wrapper width={width} height={height}>
```

Comment:
`width` and `height` are invalid props for `React.Fragment` right? Ideally we don't get `propType` errors in that case 🤔

Reply:
https://stackoverflow.com/questions/48705910/does-react-warn-of-additional-props-that-a-component-wasnt-expecting